### ISSUE TITLE
no-tls build fails tests

### ISSUE THREAD

**Issue by benlaurie** [openssl/openssl#1723]
```
Test Summary Report
-------------------
../test/recipes/80-test_ssl_new.t        (Wstat: 256 Tests: 18 Failed: 1)
  Failed test:  15
  Non-zero exit status: 1
Files=87, Tests=426, 72 wallclock secs ( 0.64 usr  0.08 sys + 42.15 cusr 36.44 csys = 79.30 CPU)
Result: FAIL
Failed 1/87 test programs. 1/426 subtests failed.
```

**Comment by levitte** [comment]
@benlaurie, is this still the case?

**Comment by jon-oracle** [comment]
Yes still the case for ./config no-tls
The above PR should resolve this.

**Comment by richsalz** [comment]
resolved via #2820; closing this.
